CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 ## master
 
+* Add `ActionView::SlotableV2`
+  * `with_slot` becomes `renders_one`.
+  * `with_slot collection: true` becomes `renders_many`.
+  * Slot definitions now accept either a component class, component class
+    name, or a lambda instead of a `class_name:` keyword argument.
```

Comment:
```suggestion
  * Slot definitions now accept either a component class, component class name, or a lambda instead of a `class_name:` keyword argument.
```